Example/NYTPhotoViewer-SwiftTests/NYTPhotosOverlayViewTests.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  NYTPhotosOverlayViewTests.swift
+//  NYTPhotoViewer
+//
+//  Created by Mark Keefe on 3/21/15.
+//  Copyright (c) 2015 The New York Times. All rights reserved.
+//
+
+import XCTest
+import UIKit
+
+class NYTPhotosOverlayViewTests: XCTestCase {
+
+    func testNavigationBarExistsAfterInitialization() {
+        let overlayView = NYTPhotosOverlayView()
+        XCTAssertNotNil(overlayView.navigationBar)
+    }
+    
+    func testLeftBarButtonItemSetterAffectsNavigationBar() {
+        let overlayView = NYTPhotosOverlayView()
+        let leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: UIBarButtonSystemItem.Done, target: nil, action: nil)
+        overlayView.leftBarButtonItem = leftBarButtonItem
+        XCTAssert(leftBarButtonItem == overlayView.navigationBar.items.first!.leftBarButtonItem)
```

Comment:
If `first` is an optional, why force unwrap it? Why not `overlayView.navigationBar.items.first?.leftBarButtonItem`?